client/src/components/FeedbackForm/FeedbackButton.module.scss

```
@@ -0,0 +1,7 @@
+.button {
```

__Consistency__: (CSS) Use standard top-level class name for CSS modules (`.container`, discrepancies will be fixed).

```suggestion
.container {
```